data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container

```diff
@@ -8,7 +8,7 @@ After=network-online.target ironic-volume.service build-ironic-env.service image
 [Container]
 ContainerName=baremetal-operator
 Image=$METAL3_BAREMETAL_OPERATOR_IMAGE
-Exec=/baremetal-operator -build-preprov-image -webhook-port 0
+Exec=/baremetal-operator -build-preprov-image -webhook-port 0 -enable-leader-election -namespace openshift-machine-api -leader-election-release-on-cancel
```

Comment:
where is `-leader-election-release-on-cancel defined`? it's not near https://github.com/openshift/baremetal-operator/blob/master/main.go#L139-L141

Reply:
Ah, it's in https://github.com/openshift/baremetal-operator/pull/353